Adyen/Helpers/UIViewHelpers.swift

```diff
@@ -19,4 +20,19 @@ extension AdyenScope where Base: UIView {
 
         return image
     }
+
+    /// :nodoc:
+    public func getTopMostView<T>() -> T? {
```

Comment:
This is breadth search, not deep search.. `TopMost` is confusing naming 🤓

Reply:
well, breadth search will get the top most view, but depth first search will return the deepest view first, hence the `topMost` in the name, why is it confusing?.

Reply:
If you have (this is just a weird en example 😅):

```
[UIView1] - [UIStackView] - [UIView3] - [UISctollView3] - [UIView4]
               |
           [UISctollView2] - [UIView5]
```

So UISctollView3 is "topMost", but this method will return UISctollView2.

It is not a problem or something, just saying the name is not describing what it does :D 
Should be `getFirstViewOfType<T>` or change `queue.removeFirst()` -> `queue.removeLast()`